src/test/java/test/sample/JUnitSample4.java

```diff
@@ -1,8 +1,8 @@
 package test.sample;
 
-import junit.framework.Assert;
 import junit.framework.TestCase;
 import junit.framework.TestSuite;
+import org.junit.Assert;
```

Comment:
Be careful, if you change the import, it is not the same test anymore :)

Reply:
I removed it, because that other one was deprecated. Should I revert it ?

Reply:
The other one is deprecated but the test is testing support of this JUnit version. Another test could exist with the use of the new import.

Reply:
@juherr - Alright! I will revert this one back.